### ISSUE TITLE
Syntax errors in examples/keras_parikh_entailment

### ISSUE THREAD

**Issue by pspiegelhalter** [explosion/spaCy#623]
`examples/keras_parikh_entailment/keras_decomposable_attention.py` gives syntax errors on Lines 104 and 120.

Suggested quickfix:

```
def _outer(AB):
    att_ji = T.batched_dot(AB[1], AB[0].dimshuffle((0, 2, 1)))
    return att_ji.dimshuffle((0, 2, 1))
```

```
def _normalize_attention(att, mat):
    att=attmat[0]
    mat=attmat[1]
    if transpose:
        att = att.dimshuffle((0, 2, 1))
    # 3d softmax
    e = K.exp(att - K.max(att, axis=-1, keepdims=True))
    s = K.sum(e, axis=-1, keepdims=True)
    sm_att = e / s
return T.batched_dot(sm_att, mat)
```

**Comment by honnibal** [comment]
Hm! Thanks.

I made some quick edits to clean things up, and haven't rerun the experiments because I didn't want to spin up another GPU server. Do you want to make a pull request? Otherwise I'm happy to apply the patch.

**Comment by honnibal** [comment]
There are a number of other small errors here. Fixing.

**Comment by Timopheym** [comment]
@honnibal Sorry i have another issue with this example. It's not a bug - so i don't want to open new one.

https://github.com/explosion/spaCy/blob/master/examples/keras_parikh_entailment/__main__.py#L16 - here we pass path to our model, but we do not update it white training. How can we do this each n epoch?
And how we can export model from keras that can be loaded to spacy? (sorry for dumb questions =))

**Comment by honnibal** [comment]
I think this should be addressed now, by #955 --- sorry for the difficulties! I've found it very tricky to get this example right and keep it up to date.
